src/common/options/global.yaml.in

```diff
@@ -4290,6 +4290,24 @@ options:
     that this recommendation may change in the future however.
   default: true
   with_legacy: true
+- name: bluefs_enable_cache
+  type: bool
+  level: advanced
+  desc: Enables caching for bluefs reads to improve performance by reducing disk reads
```

Comment:
`desc` can be even shorter, but
`long_desc` could describe relation with bluefs_buffered_io.